setup.py

```diff
@@ -50,11 +50,12 @@ def run(self):
         Extension(
             name=f'natcap.invest.{package}.{module}',
             sources=[f'src/natcap/invest/{package}/{module}.pyx'],
-            include_dirs=[numpy.get_include()],
+            include_dirs=[numpy.get_include()] + ['src/natcap/invest/managed_raster'],
```

Comment:
I'm not completely sure why, but without this, we get errors that `'LRUCache.h' file not found`